### PR TITLE
Update logs mount_windows.go

### PR DESCRIPTION
**Pull request by gaoguangze111**
Migrate to structured logs

File modified:
	staging/src/k8s.io/mount-utils/mount_windows.go

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Migrate to structured logs according to keps/sig-instrumentation/1602-structured-logging

**Which issue(s) this PR fixes**:

Fixes #
	staging/src/k8s.io/mount-utils/mount_windows.go

Ref:

    keps/sig-instrumentation/1602-structured-logging
    Structured Logging migration instructions

### CODE REVIEW

File: staging/src/k8s.io/mount-utils/mount_windows.go [hunk 1]
```diff
@@ -210,7 +209,7 @@ func removeSMBMapping(remotepath string) (string, error) {
 
 // Unmount unmounts the target.
 func (mounter *Mounter) Unmount(target string) error {
-	klog.V(4).Infof("azureMount: Unmount target (%q)", target)
+	klog.V(4).InfoS("azureMount: Unmount target", "target", target)
```
Comment (andyzhangx) [inline staging/src/k8s.io/mount-utils/mount_windows.go:212]: what's the output if `target="123"`?

**Comment by jsturtevant** [comment]
/ok-to-test
/priority important-longterm

@gaoguangze111 could you add a release note?

File: staging/src/k8s.io/mount-utils/mount_windows.go [hunk 2]
```diff
@@ -74,7 +74,7 @@ func (mounter *Mounter) MountSensitive(source string, target string, fstype stri
 	sanitizedOptionsForLogging := sanitizedOptionsForLogging(options, sensitiveOptions)
 
 	if source == "tmpfs" {
-		klog.V(3).Infof("mounting source (%q), target (%q), with options (%q)", source, target, sanitizedOptionsForLogging)
+		klog.V(3).InfoS("mounting source to target with options", "source", source, "target", target, "options", sanitizedOptionsForLogging)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/mount-utils/mount_windows.go:77]:
```suggestion
		klog.V(3).InfoS("Mounting source to target with options", "source", source, "target", target, "options", sanitizedOptionsForLogging)
```

File: staging/src/k8s.io/mount-utils/mount_windows.go [hunk 3]
```diff
@@ -83,8 +83,7 @@ func (mounter *Mounter) MountSensitive(source string, target string, fstype stri
 		return err
 	}
 
-	klog.V(4).Infof("mount options(%q) source:%q, target:%q, fstype:%q, begin to mount",
-		sanitizedOptionsForLogging, source, target, fstype)
+	klog.V(4).InfoS("begin to mount", "source", source, "target", target, "fstype", fstype, "options", sanitizedOptionsForLogging)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/mount-utils/mount_windows.go:86]:
```suggestion
	klog.V(4).InfoS("Begin to mount", "source", source, "target", target, "fstype", fstype, "options", sanitizedOptionsForLogging)
```

File: staging/src/k8s.io/mount-utils/mount_windows.go [hunk 4]
```diff
@@ -115,7 +114,7 @@ func (mounter *Mounter) MountSensitive(source string, target string, fstype stri
 				valid, err := isValidPath(source)
 				if !valid {
 					if err == nil || isAccessDeniedError(err) {
-						klog.V(2).Infof("SMB Mapping(%s) already exists while it's not valid, return error: %v, now begin to remove and remount", source, err)
+						klog.V(2).InfoS("SMB Mapping already exists while it's not valid, now begin to remove and remount", "SMBMapping", source, "error", err)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/mount-utils/mount_windows.go:117]:
```suggestion
						klog.V(2).InfoS("SMB Mapping already exists while it's not valid, now begin to remove and remount", "SMBMapping", source, "err", err)
```

File: staging/src/k8s.io/mount-utils/mount_windows.go [hunk 5]
```diff
@@ -124,7 +123,7 @@ func (mounter *Mounter) MountSensitive(source string, target string, fstype stri
 						}
 					}
 				} else {
-					klog.V(2).Infof("SMB Mapping(%s) already exists and is still valid, skip error(%v)", source, err)
+					klog.V(2).InfoS("SMB Mapping already exists and is still valid, skip error", "SMBMapping", source, "error", err)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/mount-utils/mount_windows.go:126]:
```suggestion
					klog.V(2).InfoS("SMB Mapping already exists and is still valid, skip error", "SMBMapping", source, "err", err)
```

File: staging/src/k8s.io/mount-utils/mount_windows.go (hunk 4 above)
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/mount-utils/mount_windows.go:117]:
```suggestion
						klog.V(2).InfoS("SMB Mapping already exists while it's not valid, now begin to remove and remount", "SMBMapping", source, "err", err)
```

File: staging/src/k8s.io/mount-utils/mount_windows.go [hunk 6]
```diff
@@ -148,7 +147,7 @@ func (mounter *Mounter) MountSensitive(source string, target string, fstype stri
 		klog.Errorf("mklink failed: %v, source(%q) target(%q) output: %q", err, mklinkSource, target, string(output))
```
Comment (thxCode) [inline staging/src/k8s.io/mount-utils/mount_windows.go:147]: missed

**Pull request closed without merging**